substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVTypeRecord.java

```diff
@@ -313,6 +317,94 @@ public boolean equals(Object obj) {
         }
     }
 
+    static final class CVTypeFuncIdRecord extends CVTypeRecord {
```

Comment:
I don’t think we’ll need this (see https://github.com/oracle/graal/pull/5638#discussion_r2382022923), so I’d suggest removing it.